app/src/main/java/com/asfoundation/wallet/logging/LogReceiver.kt

```diff
@@ -0,0 +1,13 @@
+package com.asfoundation.wallet.logging
+
+interface LogReceiver {
+  companion object {
+    const val DEFAULT_TAG = "default_tag"
+    const val DEFAULT_MSG = "default_message"
+    const val DEFAULT_THROWABLE_MSG = "default_throwable_msg"
+    const val DEFAULT_THROWABLE_STATCKTRACE = "default_throwable_stacktrace"
+  }
+  fun log(tag: String?, throwable: Throwable?)
+  fun log(tag: String?, message: String?)
+  fun log(tag: String?, message: String?, throwable: Throwable?)
```

Comment:
I think we should force non-null values

Reply:
We are not forcing null values, we are only allowing null values to be inline with Android's logging methods